app/download/views.py

```diff
@@ -44,7 +44,7 @@ def download_document(service_id, document_id):
 
     return response
 
-# this endpoint mimics the url format of the current document-download-frontend links sent to the end user
```

Comment:
do we only need to remove the comment here?

Reply:
yeah, the endpoint below is fine, the comment here really should have just been a PR comment on a previous PR (explaining what this endpoint was for).